woocommerce/woocommerce-android
# [HACK] Research if CallThrottler can be implemented without assumption on 200ms

Closes: #12458 


### Description

We have had a couple of issues recently with CallThrottler, which is used for navigation. After some research, @samiuelson   [found out](https://github.com/woocommerce/woocommerce-android/issues/12458#issuecomment-2321259503) that apparently, the reason why that approach was introduce was already fixed in the navigation library itself, that's why the PR removes `CallThrottler`

### Testing information

* Try initiate multiple navigations in different parts of the app
* Notice that it works properly and only one instance of a fragment is getting open

### The tests that have been performed

The one from the testing information

### Images/gif



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
